apps/challenges/views.py

```diff
@@ -573,8 +573,22 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
         with open(join(BASE_LOCATION, unique_folder_name, yaml_file), "r") as stream:
             yaml_file_data = yaml.safe_load(stream)
     except (yaml.YAMLError, ScannerError) as exc:
+        # To get the problem description
+        if hasattr(exc, 'problem'):
+            error_description = exc.problem
+            # To capitalize the first alphabet of the problem description as default is in lowercase
+            error_description = error_description[0:].capitalize()
+        # To get the error line and column number
+        if hasattr(exc, 'problem_mark'):
+            mark = exc.problem_mark
+            line_number = mark.line+1
+            column_number = mark.column+1
+        message = (
```

Comment:
I think it'd be better to display these errors on the page below the `Submit` button on `create challenge page` rather than in notification which disappears after some time. The challenge host won't be able to fix the error if it disappears.

Reply:
done.